contrib/cirrus/setup.sh

```diff
@@ -8,14 +8,29 @@ set -e
 # expectation mismatch.
 source $(dirname $0)/lib.sh
 
-req_env_vars OS_RELEASE_ID OS_RELEASE_VER GOSRC IN_PODMAN_IMAGE
+req_env_vars OS_RELEASE_ID OS_RELEASE_VER GOSRC IN_PODMAN_IMAGE CIRRUS_CHANGE_TITLE CIRRUS_PR_DRAFT
```

Comment:
[Cirrus documentation](https://cirrus-ci.org/guide/writing-tasks/#environment-variables) does not define this variable in the non-true state. That means it could be `false`, or could be empty, and if empty, `req_env_vars` will fail. (This PR is not-draft, and CI passed, so I'm guessing `CIRRUS_PR_DRAFT` is non-empty, but can we rely on that?)

Reply:
I'll just get rid of it.